dev/TEAL.md

```diff
@@ -286,12 +305,17 @@ Asset fields include `AssetHolding` and `AssetParam` fields that are used in `as
 | `pop` | discard value X from stack |
 | `dup` | duplicate last value on stack |
 | `dup2` | duplicate two last values on stack: A, B -> A, B, A, B |
+| `dig` | duplicate value N from the top of stack |
+| `swap` | swaps two last values on stack: A, B -> B, A |
+| `select` | selects one of two values to retain: A, B, C -> A ? B : C |
```

Comment:
Maybe make more explicit: If A!=0 then push B on the stack, if A==0 then push C on the stack

Reply:
Agreed.  Putting aside what we currently do, what are you thoughts on the proper behavior when A is a byte-array? I suppose we should panic, as I think that's the behavior if we `bnz` with a string on the stack. I'll change to consistency with that choice, and document it.